public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -439,7 +439,7 @@ function ContextMenu (uiContextMenu) {
      * @param label     Current label being modified.
      */
     function setTags (label) {
-        var maxTags = 10;
+        var maxTags = 11;
```

Comment:
I just incremented this by 1 for now... is there a better change you want me to implement here?

Reply:
Nah that's probably fine for now. I don't have a good alternative yet.